src/blosc2/schunk.py

```diff
@@ -469,14 +490,13 @@ def decompress_chunk(self, nchunk, dst=None):
         Examples
         --------
         >>> import blosc2
-        >>> cparams = {'typesize': 1}
-        >>> storage = {'cparams': cparams}
-        >>> schunk = blosc2.SChunk(chunksize=11, **storage)
+        >>> schunk = blosc2.SChunk(cparams = {'typesize': 1})
```

Comment:
A (minor) note about code style: when using keyword arguments, one should not put `=` between spaces. See https://pep8.org/#other-recommendations